inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -43,6 +43,16 @@ def __init__(self, protein_pdb: str, temperature: float = 300):
         self.protein_pdb = files.resolve_path(protein_pdb)
         self.temperature = temperature
 
+    def set_small_files(self):
```

Comment:
Why having this additional function if we are just downloading one file? To construct the universe with MDAnalysis you also need the .gro file and the .tpr. Shouldn't you add those files too here?